docs/specs/zk_evm/vm_specification/compiler/instructions/evm/environment.md

```diff
@@ -12,7 +12,7 @@ is common for Yul and EVMLA representations.
 ### System Contract
 
 This information is requested a System Contract called
-[L2BaseToken](https://github.com/code-423n4/2023-10-zksync/blob/main/code/system-contracts/contracts/L2BaseToken.sol).
+[L2BaseToken](https://github.com/code-423n4/2023-10-zksync/blob/main/code/system-contracts/contracts/L2EthToken.sol).
```

Comment:
doesnt look correct

Reply:
This was needed for link check, as the codeArena is the old code arena. Alternatively the new one could be used.

Reply:
I can put this if you think it is better: 
https://github.com/code-423n4/2024-03-zksync/blob/main/code/system-contracts/contracts/L2BaseToken.sol

Reply:
I would either:
- Copy from 2024-03 branch
- Or refrain from editing docs at all in this PR